app/uk/gov/hmrc/individualincomedesstub/domain/ErrorResponse.scala

```diff
@@ -29,5 +29,10 @@ sealed abstract class ErrorResponse(
   def toHttpResponse = Results.Status(httpStatusCode)(Json.toJson(this))
 }
 
+case class ErrorInvalidRequestPayload(errorMessage: String) extends ErrorResponse(BAD_REQUEST, "INVALID_REQUEST", errorMessage)
+case class ErrorInvalidPathParameter(errorMessage: String) extends ErrorResponse(BAD_REQUEST, "INVALID_REQUEST", errorMessage)
```

Comment:
In `individuals-api` we [generalised identical errors like the above two into one](https://github.com/hmrc/individuals-api/commit/32856e1c9ce5ddb07f9fea8b0d3be97ba55cdff3#diff-1f92ad51b2c1755f090df3a491341d59R32). May be a good idea to be consistent and do that here too.